.github/workflows/_test_t5x_rosetta.yaml

```diff
@@ -78,7 +83,7 @@ jobs:
         run: |
           IMAGE="$(echo ${{inputs.T5X_IMAGE}} | sed 's/\//#/')"
           TEST_CASE_NAME=${{ matrix.TEST_NAME }}
-          JOB_NAME=rosetta-T5X-${GITHUB_RUN_ID}-${TEST_CASE_NAME}
+          JOB_NAME=${{ inputs.FW_NAME }}-${GITHUB_RUN_ID}-${TEST_CASE_NAME}
```

Comment:
I'd prefer `${{ github.run_id }}` over `${GITHUB_RUN_ID}` since the former is substituted at 'compilation' time and hence is easier for visual examination.

Reply:
Sure, since this is the case for many workflows; I can bulk search-replace in a follow up PR to minimize surface of this PR